### PR TITLE
Fixes/mark html as safe 2.7

### PR DESCRIPTION
**Pull request by czpython**
Backport of #313

### CODE REVIEW

File: djangocms_text_ckeditor/fields.py [hunk 1]
```diff
@@ -12,30 +16,71 @@
...
 
     def __init__(self, *args, **kwargs):
-        # This allow widget configuration customization from the model definition
-        if kwargs.get('configuration', False):
-            self.configuration = kwargs['configuration']
-            del(kwargs['configuration'])
+        # This allows widget configuration customization
+        # from the model definition
+        if kwargs.get('configuration', None):
+            self.configuration = kwargs.pop('configuration')
```
Comment (GaretJax) [inline djangocms_text_ckeditor/fields.py]: Same as for the original PR.

File: djangocms_text_ckeditor/fields.py [hunk 2]
```diff
@@ -12,30 +16,71 @@
...
-            text_editor_widget = TextEditorWidget(configuration=self.configuration)
+            widget = TextEditorWidget(configuration=self.configuration)
         else:
-            text_editor_widget = TextEditorWidget
-        defaults = {'widget': text_editor_widget}
+            widget = TextEditorWidget
+
+        defaults = {
+            'form_class': HTMLFormField,
+            'widget': TextEditorWidget,
```
Comment (GaretJax) [inline djangocms_text_ckeditor/fields.py]: Not `widget`?

**Comment by GaretJax** [comment]
Left some comments, LGTM otherwise.

File: djangocms_text_ckeditor/tests/test_field.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        obj = SimpleText.objects.get(pk=obj.pk)
+
+        self.assertEqual(obj.text, self.text_normal)
+
+        obj = SimpleText(text=self.text_with_iframe)
+        obj.full_clean()
+        obj.save()
+
+        escaped = ('<p>some non malicious text</p>&lt;iframe '
+                   'src="http://www.w3schools.com"&gt;&lt;/iframe&gt;')
```
Comment (cyberstar) [inline djangocms_text_ckeditor/tests/test_field.py]: please consider moving this as a class attribute to reduce boilerplate.

**Comment by cyberstar** [comment]
LGTM

**Pull request merged**